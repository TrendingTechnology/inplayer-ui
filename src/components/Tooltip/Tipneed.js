import styled from 'styled-components';

const Tipneed = styled.div`
  cursor: pointer;
`;

export default Tipneed;
