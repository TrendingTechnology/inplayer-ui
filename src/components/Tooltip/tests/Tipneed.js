import { generateSnapshots } from 'helpers';

import Tipneed from '../Tipneed';

generateSnapshots(Tipneed);
