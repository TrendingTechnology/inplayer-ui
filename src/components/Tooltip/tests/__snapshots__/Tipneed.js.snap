// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tipneed renders correctly 1`] = `
.c0 {
  cursor: pointer;
}

<div
  className="c0"
/>
`;
